lib/screen/home_screen.dart

```diff
@@ -1,15 +1,453 @@
+                          ),
+                          SizedBox(height: 10),
+
+                          //(BALANCE) VALUE
+                          Container(
+                            width: double.infinity,
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.center,
+                              children: [
+                                Container(
+                                  child: Text(
+                                    'Php',
+                                    style: TextStyle(fontSize: 12),
+                                  ),
+                                ),
+                                SizedBox(width: 10),
+                                Container(
+                                  alignment: Alignment.center,
+                                  child: Text(
+                                    amountFormatter(budget(
+                                            monthlyBudgetModel.budgetList) -
+                                        expense(monthlyBudgetModel.budgetList)),
+                                    style: TextStyle(
+                                        fontSize: 30,
+                                        fontWeight: FontWeight.bold),
+                                  ),
+                                ),
+                              ],
+                            ),
+                          ),
+                          SizedBox(height: 5),
+                          Container(
+                            child: Text(
+                              'Balance',
+                              style: TextStyle(fontSize: 12),
+                            ),
+                          ),
+                          SizedBox(height: 20),
+                        ],
+                      ),
+                    ),
+
+                    Expanded(
+                      child: Container(
+                        padding: EdgeInsets.zero,
+                        margin: EdgeInsets.zero,
+                        child: ListView.builder(
+                            itemBuilder: (context, index) {
+                              return BudgetItem(
+                                index: index,
+                                budget: monthlyBudgetModel.budgetList[index],
+                                isLoading: index ==
+                                    monthlyBudgetModel.budgetList.length - 1,
+                              );
+                            },
+                            itemCount: monthlyBudgetModel.budgetList.length),
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
     );
   }
+
+  double budget(List<BudgetModelSample> items) {
+    double result = 0.0;
+    items.forEach((budget) {
+      result += budget.budget;
+    });
+    return result;
+  }
+
+  double expense(List<BudgetModelSample> items) {
```

Comment:
Don't put business log into your view logic move this logic into your view model